api/kyverno/v1/match_resources_types.go

```diff
@@ -10,7 +10,7 @@ import (
 type MatchResources struct {
 	// RequestTypes can contain values ["CREATE, "UPDATE", "CONNECT", "DELETE"], which are used to match a specific action.
 	// +optional
-	RequestTypes RequestTypes `json:"requestTypes,omitempty" yaml:"requestTypes,omitempty"`
+	Operations Operation `json:"requestTypes,omitempty" yaml:"requestTypes,omitempty"`
```

Comment:
Suggest using `admissionOperations` instead of `Operations`. Also, the `requestTypes` needs to be updated in the JSON and YAML tags.

Reply:
Sure, I will update the name right away!

> he requestTypes needs to be updated in the JSON and YAML tags.

https://github.com/kyverno/kyverno/pull/5869#issuecomment-1437323691 I am getting this error when I run `make codegen-all`

I also have trouble understanding where the logic for admissionOperations will go, and can you please tell me where the current logic for `request.operation` is at so i can understand how its implimented

Reply:
Hi @Vishal-Chdhry have you seen this logic and its usage?

https://github.com/kyverno/kyverno/blob/main/pkg/utils/match/match.go#L26